studio.eufonia.EufoniaClient.yml

```diff
@@ -0,0 +1,79 @@
+id: studio.eufonia.EufoniaClient
+
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+tags:
+  - proprietary
+
+command: eufonia-client
+finish-args:
+  - --device=dri # Hardware acceleration
```

Comment:
```suggestion
```

you have all already.

Reply:
I resolved this by removing --device=all